src/cascadia/WindowsTerminal/WindowEmperor.cpp

```diff
@@ -109,7 +109,7 @@ bool WindowEmperor::HandleCommandlineArgs()
         if (!res.Message.empty())
         {
             AppHost::s_DisplayMessageBox(res);
-            ExitThread(res.ExitCode);
+            ExitProcess(res.ExitCode);

```

Comment:
Could you call `exit(res.ExitCode)` instead? It'll correctly call any `atexit()` handlers and call `TerminateProcess` instead of `ExitProcess` for immediate shutdown.

Reply:
Are you _sure_ you want to run all our static deinitializers? ;)

Reply:
In that case I'd call `quick_exit()`. The implementation can be found at:
```
C:\Program Files (x86)\Windows Kits\10\Source\10.0.22621.0\ucrt\startup\exit.cpp
```

Reply:
There's also `_Exit()` btw, which is another proper C standard function. Using OS primitives here doesn't do us any favors as it breaks app verifier and other debugging tools.